compatibility_server/test_pip_checker.py

```diff
@@ -21,19 +21,120 @@
 import json
 import mock
 import os.path
+import subprocess
 import unittest
 
 import pip_checker
 
 
+def timestamp_to_seconds(timestamp):
+    """Convert a timestamp string into a microseconds value
+    :param timestamp
+    :return time in microseconds
+    """
+    from datetime import datetime
```

Comment:
Imports should be at the top level and should only include modules or packages.

Reply:
Done.